core/common/src/main/resources/alluxio-default.properties

```diff
@@ -33,6 +33,7 @@ alluxio.underfs.glusterfs.mapred.system.dir=glusterfs:///mapred/system
 alluxio.underfs.hdfs.configuration=${alluxio.home}/conf/core-site.xml
 alluxio.underfs.hdfs.impl=org.apache.hadoop.hdfs.DistributedFileSystem
 alluxio.underfs.hdfs.prefixes=hdfs://,glusterfs:///
+alluxio.underfs.mount.shared.publicly=false
```

Comment:
Added the documentation.
Yes, it only applies to object stores. Would you prefer to rename it to "alluxio.underfs.object.store.mount.shared.publicly" ? It seems a bit long to me ...